README.md

```diff
@@ -292,6 +293,14 @@ same major line. Should you need to upgrade to a new major, use an explicit
   project. This means that it will always use the system-wide package manager
   regardless of what is being specified in the project's `packageManager` field.
 
+- `COREPACK_ENV_FILE` can be set to `0` to request Corepack to not attempt to
+  load `.corepack.env`; it can be set to a path to specify a different env file.
+  Only keys that starts with `COREPACK_` will be taken into account, not all
+  keys that start with `COREPACK_` will be taken into account (
+  `COREPACK_ENABLE_DOWNLOAD_PROMPT` and `COREPACK_ENV_FILE` are ignored).
```

Comment:
This would be worth a test, especially for `COREPACK_ENABLE_DOWNLOAD_PROMPT` (assuming you omitted it for security reasons?)

Reply:
Good call (yes, btw I’m happy to add more which we think could be security sensitive)